OpenOversight/app/main/views.py

```diff
@@ -331,6 +331,7 @@ def officer_profile(officer_id):
             .all()
         )
         assignments = Assignment.query.filter_by(officer_id=officer_id).all()
```

Comment:
We want the department work history ("Unit") to link from the officer page (what you did is actually a very useful addition, though). 

I'm not editing the code directly cause I have no idea what I'm doing & you do. Would that modification just be changing 'assignments' & 'Assignment' here to the variable for unit or Units?  (Would love to keep what you've got also, though)